src/objects/zcl_abapgit_object_sfpf.clas.abap

```diff
@@ -113,6 +116,10 @@ CLASS ZCL_ABAPGIT_OBJECT_SFPF IMPLEMENTATION.
     TRY.
         li_wb_form = load( ).
         li_fp_form ?= li_wb_form->get_object( ).
+
+        mo_files->add_raw( iv_ext = c_layout_file_ext
+                           iv_data = li_fp_form->get_layout( )->get_layout_data( ) ).
+
```

Comment:
I would guess also a clear of the existing/old field is needed?

Reply:
I had this in the code:

```abap
li_fp_form->get_layout( )->set_layout_data( i_layout_data = conv #('') ).
```

But i removed it because it did not save any space in our SFPF XML files (because there
are serialized layouts in all other languages) and the statement instantly removes
the layout from caches and breaks the form and confuses abapGit.

So, I tried to do this:

```abap
        li_fp_layout->set_layout_data( i_layout_data = conv #( '' ) ).
        rv_xstr = cl_fp_helper=>convert_form_to_xstring( li_fp_form ).
        li_fp_layout->set_layout_data( i_layout_data = lv_layout_data ).
```

But I started getting the error `SFPF serialization error: Error when transformingobject data` in my 2nd testing system - so I guess it is not OK to set layout data to empty xstring in all systems and hence I would rather keep abapGit creating redundant data but working reliably in all systems